src/string.c

```diff
@@ -14,4 +16,23 @@ size_t strlen(const char *s) {
   return length;
 }
 
+int strcmp(const char *s1, const char *s2) {
+  printf("zoo");
+  printf("%c, - %c \n", *s1, *s2);
+
+  char string_one = *s1;
+  char string_two = *s2;
+
+  while (string_one != 0 || string_two != 0) {
```

Comment:
```suggestion
  while (string_one != 0 && string_two != 0) {
```
If you use `||` you will continue to loop after you have reached the end of the shortest string.